stories/demos/components/Editors/SelectEditor.tsx

```diff
@@ -27,13 +20,13 @@ export function SelectEditor({
       styles={{
         control: (provided) => ({
           ...provided,
-          height: rowHeight - 1,
+          height: 'var(--row-height)',
```

Comment:
```suggestion
          height: 'calc(var(--row-height) - 1)',
```

![image](https://user-images.githubusercontent.com/1452717/119183671-0a620e80-ba3a-11eb-85c6-b4886bb10632.png)

We can probably remove this editor completely